src/odemis/util/test/dataio_test.py

```diff
@@ -223,6 +223,60 @@ def test_splitext(self):
             self.assertEqual(ao, eo, "Unexpected output for '%s': %s" % (inp, ao))
 
 
+class TestSplitPlanes(unittest.TestCase):
+    def _input_image_2d(self):
+        metadata = {model.MD_SW_VERSION: "1.0-test",
+                    model.MD_HW_NAME: "fake hw",
+                    model.MD_DESCRIPTION: "sem",
+                    model.MD_DIMS: "XY"}
```

Comment:
Can the metadata not be saved as a class attribute? (except the DIMS which can be added/updated in the function)
That also allows to check if the metadata matches with the class atribute and prevents hard coding the same metadata twice.